SimpleXML/SimpleXML.php

```diff
@@ -23,10 +23,11 @@ class SimpleXMLElement implements Traversable {
 
 	/**
      * Provides access to element's children
+     * @access private Method not callable directly, stub exists for typehint only
      * @param $name child name
```

Comment:
There must be `string` before `$name`, because phpdoc assumes `$name instanceof child::class`